include/SFML/Audio/SoundStream.hpp

```diff
@@ -281,7 +281,8 @@ class SFML_AUDIO_API SoundStream : public SoundSource
 
     enum
     {
-        BufferCount = 3 ///< Number of audio buffers used by the streaming loop
+        BufferCount = 3,    ///< Number of audio buffers used by the streaming loop
+        BufferRetries = 2   ///< Number of retries (excluding initial try) for onGetData()
```

Comment:
> Currently BufferCount and BufferRetries have different values so this is not an issue, but if we wanted to change one of them that could be an issue

How would that be an issue? You can have duplicate values in the same enum.

The real question is: do this constant need to be exposed in the header.